platform/admin/admin-configuration-configinstaller/src/main/java/org/codice/ddf/admin/configuration/ConfigurationInstaller.java

```diff
@@ -135,31 +171,38 @@ private void handleUpdate(ConfigurationEvent event) {
 
         FelixConfig felixConfig = new FelixConfig(configuration);
         String pid = felixConfig.getPid();
+
         File felixFileNew = felixConfig.getFelixFile();
         File felixFileOriginal = pidFileMap.get(pid);
 
-        // Config not tracked and will not be tracked? (1)
-        if (felixFileOriginal == null && felixFileNew == null) {
-            LOGGER.debug("Was not tracked and will not track pid {}", pid);
+        if (felixFileOriginal == null) {
+            // Config not tracked? (1)
+            if (felixFileNew == null) {
+                felixFileNew = felixConfig.generateDefaultFelixFile()
+                        .getFelixFile();
+            }
```

Comment:
I'll be honest, I don't like what I'm doing here and I'm open to suggestions. I thought: 
- Make `FelixConfig` immutable, but that doesn't make sense because it wraps a `Configuration` and _will_ mutate it if instructed to do so. 
- Stop using local variables and just use `felixConfig.getFelixFile()` everywhere but that strikes me as verbose.